setup.py

```diff
@@ -26,6 +26,7 @@
     author_email="oleg.pidsadnyi@gmail.com",
     url="https://github.com/pytest-dev/pytest-factoryboy",
     version=VERSION,
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
```

Comment:
Couldn't we use this instead?
```suggestion
    python_requires="~=2.7, >=3.5"
```

Reply:
Nope:

```console
$ pip install -e .
Obtaining file:///private/tmp/pytest-factoryboy
ERROR: Package 'pytest-factoryboy' requires a different Python: 3.7.4 not in '~=2.7, >=3.5'
```